src/targz_sizes.c

```diff
@@ -120,8 +126,24 @@ int main(int argc, char** argv) {
         }
     }
 
+    unsigned int max_filename_blocks;
+    if ( LEVEL_IS_TEST( verbosity ) ) {
```

Comment:
Formatting: avoid spaces right after `(` and right before `)` to make it consistent with the rest of this file.